### DISCUSSION TITLE
error

### DISCUSSION THREAD

**Discussion by WuhaiD** [deepmodeling/reacnetgenerator#1709] [Q&A]
i have found perplexing error.i use reanetgenerator to analysis my DUMP.lammptrj file and it normaly run and skip to edge browser.But in the interface,all of the species consist of N.The element order that i‘ve set is N C H O, but it seemly only indentify N.
if i choose BONDs FILE as the input file,the phnonemenon was gone.WHY?By the way,the file which i used is interruptive.Because the simulation had ever interrupted，i choose the former as input.

![e650429a0ff20dd8a54dd4e560ed4e3](https://user-images.githubusercontent.com/113823777/190892162-68458dba-8f95-4981-8a9c-bd60d350e8e3.jpg)

**Comment by njzjz** [comment]
Are you sure you provide right element order which is corresponding to that in the input file?

**Comment by WuhaiD** [comment]
YES，but when i used bonds file as input file smoothly runing ，I‘ve tried several times. All of the results is normal and the right element order is N C H O
Maybe because of my interruptive dump file ？






------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "tongzhugroup/reacnetgenerator"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年9月18日(星期天) 下午4:17
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [tongzhugroup/reacnetgenerator] error (Discussion #1709)





 
Are you sure you provide right element order which is corresponding to that in the input file?
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you authored the thread.Message ID: ***@***.***&gt;


从QQ邮箱发来的超大附件

reax.trj3000k (110.64M, 2022年10月18日 16:22 到期)进入下载页面：http://mail.qq.com/cgi-bin/ftnExs_download?t=exs_ftn_download&k=243730357ead769b15af25274464514d54000507015c55504b56510c5a49575a56031d0c5455534f005402535b53575151525207626963100356481b161609515607005e6259&code=f705bdcb

**Reply by njzjz** [reply]
Indeed these N atoms are more like C atoms. You may check again:
(1) The data file for read_data has the order of N C H O;
(2) The atomic masses are in the order of N C H O;
(3) The command `pair_style reaxff` has the order N C H O. See https://docs.lammps.org/latest/pair_reaxff.html;
(4) N C H O is input for the ReacNetGenerator command line interface.
